operator/watches.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-- version: v1alpha1
+- version: v1
```

Comment:
This is going to cause any new operator build to not install Kiali if the user is using v1alpha1 version of Kiali CR. I think we want to duplicate this so it watches for both v1alpha1 and v1. I'm not 100% sure because I have not done this kind of thing yet with operators, but I think that is what we need to do (especially since the CRD now has "served=true" for both v1alpha1 and v1 versions - that means a user could have v1alpha1 on the system).

I will test this kind of thing and report back - I don't know how the operator SDK deals with watching for multiple CRDs and how that affects the watches.yaml.

Reply:
AFAIU, even if you have different versions of the CRD, when you query the API, it fetches the one that has `storage: true` but it just changes the `apiVersion` header. So, watching both versions would be like watching twice the same thing. At least, this is what I understand.